pkg/cloudprovider/vsphere/vsphere_test.go

```diff
@@ -58,105 +47,6 @@ bGWObufAR8Ni4QIgWpILjW8dkGg8GOUZ0zaNA6Nvt6TIv2UWGJ4v5PoV98kCIQDx
 rIiZs5QbKdycsv9gQJzwQAogC8o04X3Zz3dsoX+h4A==
 -----END RSA PRIVATE KEY-----`
 
-func configFromEnv() (cfg Config, ok bool) {
```

Comment:
Looks like these configFrom* methods can stay in vsphere_test.go rather than move to node_manager_test.go.. They are also used in vsphere_test.go and that seems like a better place for the general purpose test helper functions.

Reply:
I think I know what I did... I was testing just the unit tests within nodemanager_test.go and it was complaining that the methods were missing. Will move them back.